app/src/main/java/com/stevesoltys/seedvault/settings/SettingsFragment.kt

```diff
@@ -151,6 +151,7 @@ class SettingsFragment : PreferenceFragmentCompat() {
         setAutoRestoreState()
         lifecycleScope.launch { setMenuItemStates() }
 
```

Comment:
Commenting on a newline since github doesn't allow commenting on comment messages.

What if you squashed this commit and the initial change adding `hasInternet()` into one.